addons/web/static/src/js/views/list/list_renderer.js

```diff
@@ -315,6 +315,7 @@ var ListRenderer = BasicRenderer.extend({
             data: record.data,
             escape: true,
             isPassword: 'password' in node.attrs,
+            digits: node.attrs.digits ? JSON.parse(node.attrs.digits) : false,
```

Comment:
Maybe:

`digits: node.attrs.digits ? JSON.parse(node.attrs.digits) : undefined,`

Reply:
looks reasonable, I'll wait for @kebeclibre so if he has some suggestion too, I'll merge them